examples/galton.jl

```diff
@@ -49,31 +49,32 @@ function draw_pins(n, xpos, ypos, offset; color = "white")
 end
 
 """
-    flip_coin()
+    flip_coin(p)
 
-Simulates flipping a coin, returning -1 and 1 with equal probability. Helper function used in `move_ball`.
+Simulates flipping a coin, returning -1 with probability `p` and 1 with probability (1 - p). Helper function
```

Comment:
Should be the other way around. And `(1-p)` should also be in code tags.
You might want to have `flip_coin(p=0.5)` to have a reasonable default.

Reply:
Yes it should 1 with probability `p`